src/arrayMethodSort.js

```diff
@@ -4,9 +4,25 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
+    for (let i = 0; i < this.length - 1; i++) {
+      if (
+        compareFunction(this[i], this[i + 1]) === true ||
+        compareFunction(this[i], this[i + 1]) > 0
```

Comment:
The condition inside the if statement is redundant. The compareFunction should return a number less than 0, 0, or greater than 0. You only need to check if the result is greater than 0.